fractal-puppy/Engine
# Feature/ui stats panel

This PR include new inventory design, items stats, 8 new "real" items, when you equip and unequip the items the stats change and affect the player,

1. HEALTH    MAX HEALTH POINTS   1 point = 1 more HP
2. DEXTERITY MOVEMENT SPEED      10 points = +5% Ms
3. STRENGTH  ATTACK DAMAGE       attack = Actual points * 1/10
4. MANA      MAX MANA            1 point = 1 more MANA

Steps to test:

- Recompile everything.
- Load scene Level0-Graveyard
- Play the game.
- Loot items (by click them standing right up the item)
- Equip / Unequip items and check changes
